test/integration/linux-5.10.11/data-reloc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/drivers/usb/serial/ir-usb.c b/drivers/usb/serial/ir-usb.c
+index 172261a908d8..4fdd666cc75a 100644
+--- a/drivers/usb/serial/ir-usb.c
++++ b/drivers/usb/serial/ir-usb.c
+@@ -194,10 +194,15 @@ static u8 ir_xbof_change(u8 xbof)
+ 	return(result);
+ }
+ 
++#include "usb-wwan.h"
+ static int ir_startup(struct usb_serial *serial)
+ {
+ 	struct usb_irda_cs_descriptor *irda_desc;
+ 	int rates;
++	volatile int i = 0;
++	static volatile void *funcs[] = {usb_wwan_open, usb_wwan_close};
++
++	printk("kpatch: usb_wwan_open=%p\n", funcs[i]);
+ 
+ 	irda_desc = irda_usb_find_class_desc(serial, 0);
+ 	if (!irda_desc) {
```

Comment:
I can't seem to build this patch, I get this error,
```
ERROR: ir-usb.o: 1 function(s) can not be patched
ir-usb.o: WARNING: unable to correlate static local variable funcs.20 used by ir_startup, assuming variable is new
ir-usb.o: function irda_usb_dump_class_desc.isra.0 doesn't have patchable function entry, unable to patch
/root/github-kpatch/kpatch-build/create-diff-object: unreconcilable difference
```

Here are my versions, probably something is compiling slightly different for me:
gcc: gcc version 11.2.1 20220127 (Red Hat 11.2.1-9) (GCC)
ld: GNU ld version 2.35.2-17.el9
kernel tree: amazon-arm-livepatch-v5.10
kpatch-build: surajjs95-arm64

Reply:
OK, I'm not ready for gcc11 environment yet. I will check it as soon as it is ready.

Reply:
I could not get with gcc11 of ubuntu21.04.
I can prepare rhel-9 at my workplace, so I will try it this week.

my versions:
gcc (Ubuntu 11.2.0-7ubuntu2) 11.2.0
ld (GNU Binutils for Ubuntu) 2.37
kernel tree: https://github.com/sm00th/linux/tree/kpatch/aarch64
kpatch-build: https://github.com/joe-lawrence/kpatch/tree/arm64-rebase